content/references/data-api.md

```diff
@@ -2824,48 +2802,40 @@ Response:
-  "marker": "n949f75evCHwgyP4fPVgaHqNHxUVN15PsJEZ3B3HnXPcPjcZAoy7|20160608020910|4499E5D60FD4F3F239C5B274E65B9CAE033398BE976E4FB49E9B30508D95F5A5",
-  "validations": [
-    {
-      "count": 27,
-      "first_datetime": "2016-06-08T02:09:21.280Z",
-      "last_datetime": "2016-06-08T02:09:21.390Z",
-      "ledger_hash": "246C5142A108C7B64F5D700936D31360B7790FA6349A98A2A1F7A14671D70B48",
-      "reporter_public_key": "n9KKTtooV4h2UsmNEhBqPvMRNj6RAHLPS6Baktf4u1AhgKNbyJAF",
-      "signature": "304402204506DD69B831886C4738F97CEED43AAFDBA67254D3EAF4FB5B3BF167A15B1B690220147E366CA53A3EC8B513978F2BA3A0905DC412CCE553E8ECE544236B097F2C8A",
-      "validation_public_key": "n949f75evCHwgyP4fPVgaHqNHxUVN15PsJEZ3B3HnXPcPjcZAoy7"
-    },
-    {
-      "count": 27,
-      "first_datetime": "2016-06-08T02:09:17.275Z",
-      "last_datetime": "2016-06-08T02:09:17.383Z",
-      "ledger_hash": "1DC9245CDBFE2C640ACD766DB4AF1DE66F6E92A8EE78F628281A6568760DBAB2",
-      "reporter_public_key": "n9JySgyBVcQKvyDoeRKg7s2Mm6ZcFHk22vUZb3o1HSosWxcj9xPt",
-      "signature": "3045022100E07D8CB801EC7AC98DB1DED813D49AE1FFE3C4CB027314EB6ED1BA7796653DE902204E65E96B0961AC09D8D7542EC59B3CE2ECAE6BC02557A4D1C0385DED00445329",
-      "validation_public_key": "n949f75evCHwgyP4fPVgaHqNHxUVN15PsJEZ3B3HnXPcPjcZAoy7"
-    },
+  "count": 141,
+  "validators": [
     {
-      "count": 27,
-      "first_datetime": "2016-06-08T02:09:13.277Z",
-      "last_datetime": "2016-06-08T02:09:13.387Z",
-      "ledger_hash": "0E11FA2052E8D345069CF09F13D76A8EF618C32F2B25A948FF104D59F53371BE",
-      "reporter_public_key": "n9KKTtooV4h2UsmNEhBqPvMRNj6RAHLPS6Baktf4u1AhgKNbyJAF",
-      "signature": "3045022100C1252F3FE8D0683F7FE5261D36876650EE185EEFE558150DEEF67F86E928463F022017237E1D89D5DBD8C4C2CC041C2EA73D96260D44A283D6CFF95359E86008E765",
-      "validation_public_key": "n949f75evCHwgyP4fPVgaHqNHxUVN15PsJEZ3B3HnXPcPjcZAoy7"
+      "validation_public_key": "nHBidG3pZK11zQD6kpNDoAhDxH6WLGui6ZxSbUx7LSqLHsgzMPec",
+      "domain: "bitso.com",
+      "chain": "main",
+      "current_index": 42281151,
+      "agreement_1h": {
+        "missed": 0,
+        "total": 1029,
+        "score": "1.0000",
+        "incomplete": false
+      },
+      "agreement_24h": {
+        "missed": 0,
+        "total": 23585,
+        "score": "1.0000",
+        "incomplete": false
+      },
+      "partial": false,
+      "unl": true
     }
   ]
 }
 ```
 
 
 
-## Get Validations
-[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routes/network/getValidations.js "Source")
+## Get Validator Manifests
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routes/network/getManifests.js "Source")
 
-Retrieve validation votes, including votes for ledger versions that are outside the main ledger chain. _(New in [v2.2.0][])_
+Retrieve manifests signed by a specified validator, used to designate ephemeral key to sign proposals and validations. _(New in [v2.3.7][])_
```

Comment:
The `[v2.3.7][]` link isn't showing up in the generated HTML. Sorry if I'm pointing out stuff that you were already planning to resolve! Just mentioning it to be safe. 🙌

Reply:
Nah, good catch. And changed it to "an ephemeral key" (one at a time)